service/src/main/kotlin/fi/espoo/evaka/shared/dev/DataInitializers.kt

```diff
@@ -1178,14 +1181,23 @@ data class DevPedagogicalDocument(
     val id: PedagogicalDocumentId,
     val childId: ChildId,
     val description: String,
+    val createdBy: EvakaUser =
```

Comment:
We need the user's name in the UI, hence sending back the entire EvakaUser object in PedagogicalDocument. Shouldn't the DevPedagogicalDocument match the actual PedagogicalDocument class exactly? I've updated the inserts to use the id, just as I did in the previous set of changes.

Reply:
I don't think it needs to match exactly, otherwise we wouldn't even need two separate classes. 

The Dev-classes are representations of what to insert into DB as test data. PedagogicalDocument is a representation that is read from DB and returned from a GET endpoint. They share many similarities, but they aren't identical. 

In this test data insert we only care about the user id and assume that a user with such id has already been inserted. Therefore DevPedagogicalDocument should not include e.g. user's name through the EvakaUser class, because it's never used. If you can pass the user's name to DevPedagogicalDocument, then one could assume that it also inserts the user with that name, which it does not do.